### PR TITLE
Output timing information to logs in applyMigrations

### PR DESCRIPTION
**Pull request by tomhoule**
- How long did the analysis step take?
- How long did applying each migration take?
- How long did applying migrations in total take?

Logged at the `INFO` level, so it is readable in the CLI `DEBUG=*`
output by default.

closes https://github.com/prisma/prisma/issues/10106

### CODE REVIEW

File: migration-engine/core/src/commands/apply_migrations.rs [hunk 1]
```diff
@@ -91,6 +97,12 @@ where
                 migration_persistence.record_successful_step(&migration_id).await?;
                 migration_persistence.record_migration_finished(&migration_id).await?;
                 applied_migration_names.push(unapplied_migration.migration_name().to_owned());
+                let migration_duration_ms = Instant::now().duration_since(migration_apply_start).as_millis() as u64;
+                tracing::info!(
+                    migration_duration_ms = migration_duration_ms,
```
Comment (tomhoule) [inline migration-engine/core/src/commands/apply_migrations.rs:102]: could be just `migration_duration_ms,`

**Comment by janpio** [comment]
Can you maybe add an example of the output to the PR description? Thanks

**Review by pimeys — APPROVED** [review]
Should we also extend this to other commands?

**Comment by tomhoule** [comment]
If we decide to put timers everywhere, I think we should but a bit more thought in the design. Maybe rely on tracing spans (it should be relatively easy) with a specific attribute.

**Comment by Jolg42** [comment]
I can now see this with `DEBUG="prisma:migrate*" npx prisma@dev migrate dev --skip-generate` (dev = 3.6.0-dev.39 here)

```
  prisma:migrateEngine:rpc SENDING RPC CALL {"id":2,"jsonrpc":"2.0","method":"applyMigrations","params":{"migrationsDirectoryPath":"/Users/j42/Dev/prisma-meow/packages/migrate/fixtures/blog/migrations"}} +11ms
  prisma:migrateEngine:stderr {"timestamp":"2021-11-29T14:57:14.051817+00:00","level":"INFO","fields":{"message":"Analysis run in 1ms","analysis_duration_ms":1},"target":"migration_core::commands::apply_migrations","span":{"name":"ApplyMigrations"},"spans":[{"name":"ApplyMigrations"}]} +6ms
  prisma:migrateEngine:stderr {"timestamp":"2021-11-29T14:57:14.051892+00:00","level":"INFO","fields":{"message":"Applying `20211012142017_`","script":"-- CreateTable\nCREATE TABLE \"Blog\" (\n    \"id\" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,\n    \"viewCount20\" INTEGER NOT NULL\n);\n"},"target":"migration_core::commands::apply_migrations","span":{"migration_name":"20211012142017_","name":"Applying migration"},"spans":[{"name":"ApplyMigrations"},{"migration_name":"20211012142017_","name":"Applying migration"}]} +1ms
  prisma:migrateEngine:stderr {"timestamp":"2021-11-29T14:57:14.052521+00:00","level":"INFO","fields":{"message":"Applying migration `20211012142017_`","migrate_action":"log"},"target":"sql_migration_connector::sql_database_step_applier","span":{"migration_name":"20211012142017_","name":"apply_script"},"spans":[{"name":"ApplyMigrations"},{"migration_name":"20211012142017_","name":"Applying migration"},{"migration_name":"20211012142017_","name":"apply_script"}]} +0ms
Applying migration `20211012142017_`
  prisma:migrateEngine:stderr {"timestamp":"2021-11-29T14:57:14.054313+00:00","level":"INFO","fields":{"message":"Migration executed in 2ms","migration_duration_ms":2},"target":"migration_core::commands::apply_migrations","span":{"migration_name":"20211012142017_","name":"Applying migration"},"spans":[{"name":"ApplyMigrations"},{"migration_name":"20211012142017_","name":"Applying migration"}]} +2ms
  prisma:migrateEngine:stderr {"timestamp":"2021-11-29T14:57:14.054335+00:00","level":"INFO","fields":{"message":"All the migrations executed in 2ms","apply_migrations_duration_ms":2},"target":"migration_core::commands::apply_migrations","span":{"name":"ApplyMigrations"},"spans":[{"name":"ApplyMigrations"}]} +0ms
```

**Pull request merged**